lib/web/apiserver.go

```diff
@@ -547,8 +546,19 @@ func (h *Handler) oidcLoginWeb(w http.ResponseWriter, r *http.Request, p httprou
 	if connectorID == "" {
 		return nil, trace.BadParameter("missing connector_id query parameter")
 	}
+
+	csrfToken, err := csrf.ExtractTokenFromCookie(r)
+	if connectorID == "" {
```

Comment:
Seems wrong. Did you mean `err != nil` maybe? Or `csrfToken == ""`? In any case, the error shouldn't be ignored as well.